### DISCUSSION TITLE
Revisar la posibilidad de cambiar dataTables por reactTable

### DISCUSSION THREAD

**Discussion by rafaelposadaf** [unix4you2/practico#25] [Ideas]
Dentro de las opciones estan estas, es posible que existan mas:
https://www.material-react-table.com/docs/examples/minimal
https://mui.com/material-ui/react-table/

**Comment by unix4you2** [comment]
Aquí el tema es que absolutamente todas las funcionalidades definidas para los reportes bajo datatables deberían tener su equivalente en reactTable y así permitir al desarrollador seleccionar el tipo de tabla a generar.
PERO! habría que revisar si además los reactTables requieren cargue de otras clases asociadas a Material, o si funciona de manera autónoma.

![image](https://github.com/unix4you2/practico/assets/2445412/05269f3b-9f66-4ea9-a98f-a1426fe3079c)

En resumen si queremos contar con esto se tendría que revisar no sólo cada equivalencia sino también el framework, si bootstrap es o no compatible.

Crearías una rama para esta prueba?   Si es así me cuentas para seguirla.
